docs/src/usage/results.md

```diff
@@ -0,0 +1,48 @@
+# Loading Results
+
+## Loading ReefModEngine Results
+
+Results from ReefModEngine.jl can be loaded with the `load_results` function. There are two
+options depending on the location of the results and accompanying data files. 
```

Comment:
You state "There are two options" but only mention show one? What is the other option?